safe_transaction_service/history/tests/utils.py

```diff
@@ -32,3 +36,9 @@ def just_test_if_mainnet_node() -> str:
             )
     just_test_if_mainnet_node.checked = True
     return mainnet_node_url
+
+
+def get_blocks_processed(indexer: EventsIndexer, ethereum_client: EthereumClient):
```

Comment:
I think this function does not make sense. In the tests you should state how many blocks you are expecting to index. IMHO with this function you are not doing a good test as you are using almost the same logic of the function to test

Reply:
Done  c392a6ed93f04e5d73ce0cbf82c1d29202c60f50